packages/transactions/src/schema.ts

```diff
@@ -15,13 +15,45 @@ import {
     FullAccessPermission,
     FunctionCall,
     FunctionCallPermission,
+    SignedDelegate,
     Stake,
     Transfer,
 } from './actions';
+import { DelegateAction } from './delegated';
+import { DelegateActionPrefix } from './prefix';
+import { Signature } from './signature';
+
+/**
+ * Borsh-encode a delegate action for inclusion as an action within a meta transaction
+ * NB per NEP-461 this requires a Borsh-serialized prefix specific to delegate actions, ensuring
+ *  signed delegate actions may never be identical to signed transactions with the same fields
+ * @param delegateAction Delegate action to be signed by the meta transaction sender
+ */
+export function encodeDelegateAction(delegateAction: DelegateAction) {
+    return new Uint8Array([
+        ...serialize(SCHEMA, new DelegateActionPrefix()),
+        ...serialize(SCHEMA, delegateAction),
+    ]);
+}
+
+export function encodeTransaction(transaction: Transaction | SignedTransaction) {
+    return serialize(SCHEMA, transaction);
+}
+
+/**
+ * Borsh-decode a Transaction instance from a buffer
+ * @param bytes Buffer data to be decoded
+ */
+export function decodeTransaction(bytes: Buffer) {
+    return deserialize(SCHEMA, Transaction, bytes);
+}
```

Comment:
Initially I wanted to opportunistically refactor the Transaction and SignedTransaction classes to decouple the class definitions from SCHEMA, but ultimately decided it was too much unrelated effort for a half measure.

I opted to leave these new methods in place and use them in favor of the encode/decode class methods but they're not strictly relevant to this PR.